code/modules/mob/living/simple_animal/abnormality/he/norinori.dm

```diff
@@ -0,0 +1,221 @@
+
+//attack vars
+	var/split_cooldown
+	var/split_cooldown_time = 8 SECONDS
+
+//PLAYABLES ATTACKS
+	attack_action_types = list(
+		/datum/action/cooldown/norisplit
+	)
+
+/datum/action/cooldown/norisplit
+	name = "Split"
+	icon_icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	button_icon_state = "norinori"
+	check_flags = AB_CHECK_CONSCIOUS
+	transparent_when_unavailable = TRUE
+	cooldown_time = 8 SECONDS
+
+/datum/action/cooldown/norisplit/Trigger()
+	if(!..())
+		return FALSE
+	if(!istype(owner, /mob/living/simple_animal/hostile/abnormality/norinori))
+		return FALSE
+	var/mob/living/simple_animal/hostile/abnormality/norinori/N = owner
+	if(N.IsContained())
+		return FALSE
+	if(N.transformed)
+		N.UnTransform()
+		StartCooldown()
+		return TRUE
+	else
+		N.Transform()
+		return FALSE
+
+
+//Init
+/mob/living/simple_animal/hostile/abnormality/norinori/Initialize()
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH, .proc/On_Mob_Death) // Hell
+
+/mob/living/simple_animal/hostile/abnormality/norinori/Destroy()
+	UnregisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH)
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/norinori/proc/On_Mob_Death(datum/source, mob/living/died, gibbed)
+	SIGNAL_HANDLER
+	if(!(status_flags & GODMODE)) // If it's breaching right now
+		return FALSE
+	if(!ishuman(died))
+		return FALSE
+	if(!died.ckey)
+		return FALSE
+	datum_reference.qliphoth_change(-1) // One death reduces it
+	return TRUE
+
+//Work
+/mob/living/simple_animal/hostile/abnormality/norinori/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time)
+	if(work_type == ABNORMALITY_WORK_ATTACHMENT && !(GODMODE in user.status_flags))
+		KillCheck(user, TRUE)
+
+//Pet it as a clerk, die.
+/mob/living/simple_animal/hostile/abnormality/norinori/funpet(mob/petter)
+	if(!IsContained())
+		return
+	if(get_attribute_level(petter, TEMPERANCE_ATTRIBUTE) <= 60)
+		KillCheck(petter)
+	else
+		datum_reference.qliphoth_change(-1)
+
+/mob/living/simple_animal/hostile/abnormality/norinori/proc/KillCheck(mob/living/carbon/human/user)
+	if(!IsContained())
+		return
+	icon_state = "norinori_transformed"
+	playsound(src, 'sound/effects/blobattack.ogg', 150, FALSE, 4)
+	playsound(src, 'sound/weapons/chainsawhit.ogg', 200, FALSE, 4)
+	attack_sound = 'sound/abnormalities/helper/attack.ogg'
+	user.Stun(3 SECONDS)
+	step_towards(user, src)
+	SLEEP_CHECK_DEATH(0.5 SECONDS)
+	step_towards(user, src)
+	SLEEP_CHECK_DEATH(0.5 SECONDS)
+	user.attack_animal(src)
+	SLEEP_CHECK_DEATH(0.2 SECONDS)
+	user.attack_animal(src)
+	SLEEP_CHECK_DEATH(0.5 SECONDS)
+	to_chat(user, "<span class='userdanger'>[src] mutilates you!</span>")
+	user.apply_damage(3000, RED_DAMAGE, null, user.run_armor_check(null, RED_DAMAGE), spread_damage = TRUE)
+	playsound(user, 'sound/abnormalities/helper/attack.ogg', 100, FALSE, 4)
+	attack_sound = initial(attack_sound)
+	user.gib() //update it with cool effects
```

Comment:
Hear me out,
```suggestion
	if(user.stat == DEAD)
		user.gib() //update it with cool effects
```
IF someone with max fortitude, KoD blessing, and Mimicry wants to pet the funny cat and get ALMOST Murdered, let them!